src/cpp/utils/dump/CMakeLists.txt

```diff
@@ -25,32 +30,24 @@ target_link_libraries(aiebu-dump PRIVATE aiebu_static)
 
 if (${CMAKE_SYSTEM_NAME} STREQUAL "Linux")
   target_link_libraries(aiebu-dump PRIVATE pthread)
-
-  # Upstreaming does not allow statically linked executables.  AIEBU_UPSTREAM is
-  # set by projects that are built for upstreaming and includes AIEBU as a
-  # submodule
-  if (NOT AIEBU_UPSTREAM)
```

Comment:
Please do not remove this code. This is required for upstreaming aiebu into Ubuntu and Fedora official packages.